step-1-deploy

```diff
@@ -60,11 +57,9 @@ done
 
 cd ../../
 
-# shellcheck source=/dev/null
-./base-deploy
+"$BIN_DIR"/base-deploy
 
-# shellcheck source=/dev/null
-. scripts/load-addresses
+loadAddresses
```

Comment:
Is this now an executable in `PATH`? How did it get there?

Reply:
I added it as a `bash` function in `lib/common.sh`: https://github.com/makerdao/testchain-dss-deployment-scripts/pull/32/files#diff-a4f791e8f95c01ba067d1d200d6935bbR21